.env.production

```
@@ -2,14 +2,15 @@
 NODE_ENV=production
 NEXT_PUBLIC_APP_URL=https://svlentes.com.br
 NEXTAUTH_URL=https://svlentes.com.br
+NEXTAUTH_SECRET=Kc8CIqipA+Q3kiQKWsNw8oSC/XZb0+n8JpKy9ScysEM=
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The NEXTAUTH_SECRET key should go before the NEXTAUTH_URL key

(UnorderedKey)